packages/utils/src/getWalletCurrencyIcon.ts

```diff
@@ -0,0 +1,35 @@
+const getWalletCurrencyIcon = (currency: string, is_dark_mode_on = false, is_modal = false) => {
+    switch (currency) {
+        case 'demo':
+            if (is_modal) return 'IcWalletDerivDemoLight';
+            return is_dark_mode_on ? 'IcWalletDerivDemoDark' : 'IcWalletDerivDemoLight';
+        case 'USD':
+            return 'IcWalletCurrencyUsd';
+        case 'EUR':
+            return 'IcWalletCurrencyEur';
+        case 'AUD':
+            return 'IcWalletCurrencyAud';
+        case 'GBP':
+            return 'IcWalletCurrencyGbp';
+        case 'BTC':
+            return is_dark_mode_on ? 'IcWalletBitcoinDark' : 'IcWalletBitcoinLight';
+        case 'ETH':
+            return is_dark_mode_on ? 'IcWalletEtheriumDark' : 'IcWalletEtheriumLight';
```

Comment:
There is typo here: `Ethereum` is correct spell. But the name of the icon file has typo, too. You need to change the file name, too. If it's out of this card's scope you can ignore this comment. You can just leave a `TODO` for correcting it later :pray:


```suggestion
            return is_dark_mode_on ? 'IcWalletEthereumDark' : 'IcWalletEthereumLight';
```